hs_core/models.py

```diff
@@ -4198,7 +4198,24 @@ def _create_person_element(self, etree, parent_element, person):
         if person.name:
             hsterms_name = etree.SubElement(dc_person_rdf_Description,
                                             '{%s}name' % self.NAMESPACES['hsterms'])
-            hsterms_name.text = person.name
+
+            if "," in person.name:
+                name_parts = person.name.split(",")
+                first_names = None
+                last_names = None
+                if len(name_parts) == 1:
+                    last_names = name_parts[0]
```

Comment:
This block will never execute, because "," is in person.name, you're guaranteed to get at least two items in the resulting list.